rsync_tmbackup.sh

```diff
@@ -613,8 +610,9 @@ while : ; do
 	# Add symlink to last backup
 	# -----------------------------------------------------------------------------
 
+	fn_run_cmd "mv -- '$DEST' '$DEST_DONE'"
```

Comment:
this use of `mv` can be dangerous on some systems (like on tape backups; yes these are really still used in the wild as they are very cheap and long lasting)
it might be safer (as these _are_ backups after all) to use a conflict check or `-f` (force) flag to ensure backups make it when they have to overwrite partial or corrupt copies from previous runs.

also have you tested what happened with symlink conflicts?

Reply:
I just used the same arguments for `mv` it was used before (when continuing an aborted backup) ([L485](https://github.com/laurent22/rsync-time-backup/pull/189/files/a5259ce9e4875cb702222d9a10b402abd088d6e3#diff-703e143870d3454e964014cb5854a43eL485)).
I see that `mv -T` (or with long option `mv --no-target-directory`) is safer than plain `mv`. With `-f`, it only overwrites empty directories, but does never move the directory to somewhere unexpected. Or what symlink conflict do you mean?

The perpose of this `mv` is to mark the backup as completed. It should not overwrite any data (except in very strance or constructed cases).

I do not know enough about tape backups. Can they be used with current version? How does rsync decide if a hardlink should be created without rewinding? By caching data?

Reply:
any reply @reactive-firewall ?